schedule/__init__.py

```diff
@@ -48,6 +48,9 @@
 import re
 import time
 
+logging.basicConfig(
+    format='%(asctime)-15s - %(levelname)s (%(name)s) - %(message)s'
+)
```

Comment:
```suggestion
```

`logging.basicConfig` [configures the root logger](https://docs.python.org/3/library/logging.html#logging.basicConfig). Subsequent calls to basicConfig are ignored.

With this addition, if the schedule library is imported at the top of the file, the root logger is configured using the schedule `basicConfig`. Now, if the the program itself calls `basicConfig` to configure the root logger, that config will be ignored because it has already been configured by schedule. This is an example of such:

```
import logging
import schedule

logging.basicConfig(format='CUSTOMFORMAT: %(message)s', level=logging.DEBUG)

logging.info("test")
```
I would expect this code to print `test`, but with the proposed change it doesn't print anything. 

This will be quite confusing for users. Therefore I'm proposing to remove this `basicConfig` call and merge the pr without it. Let me know what you think!